contracts/meta-aggregator/MetaAggregatorManager.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.24;
```

Comment:
We're using `pragma solidity 0.8.17;` everywhere in our contracts, please also use this version in your contracts